src/components/Mentors/Literals/listsData.js

```diff
@@ -1,57 +1,1002 @@
 const listsData = {
   states: [
```

Comment:
To ensure code readability, countries array should be added inside listData instead replacing states data with country data. Also, replacing the current states with the countries could potentially have a side effect when matching mentors with mentees.